### PR TITLE
Document the new parameter resolveArtifacts of RESOLVE_ONLY Maven rep…

### PR DESCRIPTION
**Pull request by ppalaga**
…o generation strategy

### CODE REVIEW

**Comment by ppalaga** [comment]
I have added a section about setting a Local caching proxy for Indy in bd7325b . 
@saumya1singh @harshmadhani @cunningt this is handy when generating the Maven repo zip locally on a slow & flaky connection. In this way `pig` can re-use the artifacts successfuly downloaded in the past, even though those previous `pig` invocations may have failed due to network issues or similar.

**Comment by ppalaga** [comment]
cc @svkcemk

**Review by thescouser89 — CHANGES_REQUESTED** [review]

**Comment by thescouser89** [comment]
Hi @ppalaga !

I wanted to apologize for the slow review on your 2 PRs. We've been hit by public holidays and another urgent initiative whose deadline was very close.

File: guide/configuration.md [hunk 1]
```diff
@@ -37,3 +37,177 @@ keycloak:
     # if service account
     # clientSecret: ""
 ```
+
+# Local caching proxy for Indy
```
[the remark this answers is not in the document]
Comment (ppalaga) [reply guide/configuration.md]: Thanks, done.

**Review by thescouser89 — APPROVED** [review]

**Pull request merged**